### DISCUSSION TITLE
I have been getting this error shown below when i tried to run Terraform plan: Workstation: 54.234.182.191

### DISCUSSION THREAD

**Discussion by gopi-chowdhary** [learndevopsonline/learndevopsonline#1487] [Q&A]
terraform plan getting error 
![Screenshot (5)](https://github.com/learndevopsonline/learndevopsonline/assets/145432552/5c9aa3e0-34ac-4423-896f-892e32cbede8)

**Comment by sreedharm07** [comment]
Hi @gopi-chowdhary 
attach IAM role to workstation to fetch data from aws

**Comment by gopi-chowdhary** [comment]
tq bro its working now

**Comment by vikram9491** [comment]
> Hi @gopi-chowdhary attach IAM role to workstation to fetch data from aws

hi @sreedharm07  do you have linkedin account , can i connect you ...? if possible can you share your profile name
